data/noiseprofiles.json

```diff
@@ -1201,6 +1201,17 @@
             {"name": "EOS R iso 102400", "iso": 102400, "a": [0.00109197871643545, 0.000517077848285967, 0.000844724668941281], "b": [7.8927573357425e-06, 7.52055337520828e-06, 9.80128311626769e-06]}
           ]
         },
+        {
+          "comment": "eos d60 contributed by PeterWem",
+          "model": "EOS D60",
+          "profiles": [
+            {"name": "EOS D60 iso 100", "iso": 100, "a": [1.36406478037885e-05, 4.71801288650747e-06, 4.49147092998427e-06], "b": [-1.79594608254617e-09, 4.93714256557499e-09, 3.77423652251075e-09]},
+            {"name": "EOS D60 iso 200", "iso": 200, "a": [1.64922309690471e-05, 5.93512833354441e-06, 8.71280849716513e-06], "b": [1.63736758264013e-10, 5.43488061070343e-10, 6.10413895337077e-10]},
+            {"name": "EOS D60 iso 400", "iso": 400, "a": [3.58147073617352e-05, 1.19811066798068e-05, 1.58984730041545e-05], "b": [4.61416104976025e-09, 6.17963414176452e-09, 1.31227683636844e-08]},
+            {"name": "EOS D60 iso 800", "iso": 800, "a": [6.79834799910035e-05, 2.21390988276476e-05, 3.37964490618815e-05], "b": [8.1311148938383e-09, 1.65102862462688e-08, 1.42532815290596e-08]},
+            {"name": "EOS D60 iso 1007", "iso": 1007, "a": [6.70409177756273e-05, 2.13249028535912e-05, 2.80806894890406e-05], "b": [4.95818689559605e-09, 1.39978489467886e-08, 1.19469599534896e-08]}
```

Comment:
ISO **1007**? Is that a typo?

Reply:
No, I was surprised too but that's the value reported in the exifs

Reply:
Ok, but I bet for an exif or in camera software bug. Anyway as this camera is old we'll have to live with that.